### ISSUE TITLE
fix: Accordion does not process `expand-mode` toggle properly

### ISSUE THREAD

**Issue by olaf-k** [microsoft/fast#6586]
# 🐛 Bug Report

When `expand-mode` switches:
- from `single` to `multi`, `aria-disabled` is not removed from the current open item.
- from `multi` to `single`, multiple open items stay open.

## 💻 Repro or Code Sample

https://codepen.io/olaf-k/pen/BaPLyMN

## 🤔 Expected Behavior

When `expand-mode` switches:
- from `single` to `multi`, `aria-disabled` should be removed from the current open item.
- from `multi` to `single`, only one item should remain open (could even reset to default and not necessarily be any of the currently open ones).

## 💁 Possible Solution

An `expandmodeChanged()` method can be created like this:
```javascript
public expandmodeChanged(_: AccordionExpandMode, nextMode: AccordionExpandMode) {
	if (!this.$fastController.isConnected) return;

	if (nextMode === AccordionExpandMode.multi) {
		this.findExpandedItem()?.removeAttribute('aria-disabled');
	} else {
		this.setItems();
	}
}
```
This requires `findExpandedItem` to be fixed as suggested in #6584
